### DISCUSSION TITLE
Prelabeled data - 1. ris ->csv? 2. adding labels with columns? .

### DISCUSSION THREAD

**Discussion by Hanedinges** [asreview/asreview#1468] [Q&A] [answered]
Hi there!

I am trying to understand how to use partially prelabeled data, however the info I read here https://asreview.readthedocs.io/en/latest/data_labeled.html#partially-labeled-data is not sufficient to help me get started. 

I remember reading a post yesterday that referred to adding a column in a csv file 'included' or 'excluded', but I can't find it anymore and I already got stuck when trying to create a simple csv file. 
Would someone please be so kind to help me out?

I have tried to use Rayyan for it, however, unfortunately, in the csv file Rayyan produces, the labels are not shown seperately in one column, so I would still have to create these manually. 
![afbeelding](https://github.com/asreview/asreview/assets/22755050/ba17ba8a-0192-447f-81d9-ea7bed0024c1)

Second option is: using Zotero for it. So: import everything into Zotero, label the articles for training ASReview and export the whole set. Other labels than exclude or include must be deleted manually. See screenshot
![afbeelding](https://github.com/asreview/asreview/assets/22755050/f3bbf342-73ac-4ea9-a4d6-5042e4474589)

Then filter 'include' and create a seperate column for it. This would then result in a csv file with a column 'exclude' and 'include'.

Question 1: I think this one will work. Is this the best way? Most efficient? 
Question 2: What exactly should the csv file look like in terms of: column, where should the exclude/include column be added? After all other ones?

It would be great to learn how to proceed most efficiently:) 

Kind regards, Simone

Below the info I read in the docs: 
 
Partially labeled datasets are datasets with a labeling decision for a subset of the records in the dataset and no decision for another subset.

A partially labeled dataset can be obtained by exporting results from ASReview LAB or other software. It can also be constructed given the format described above by merging a labeled dataset with new unlabeled records.

Partially labeled datasets are useful as the labels will be recognized by ASReview LAB as [Prior Knowledge](https://asreview.readthedocs.io/en/latest/project_create.html#select-prior-knowledge), and labels are used to train the first iteration of the active learning model.

**Comment by Rensvandeschoot** [comment, marked as answer]
For more details on data formatting, I recommend checking out our documentation here: https://asreview.readthedocs.io/en/latest/data_format.html. As long as you use one of the headers in the table and labels are either 0 (irrelevant), 1 (relevant), or left empty (for those to be screened), the order of the columns is not an issue.

Additionally, you might find the ASReview DataTools package helpful for creating datasets. It's a robust tool designed specifically for this purpose. You can explore it here: https://github.com/asreview/asreview-datatools.

There are also some tutorials available that can guide you on how to use the DataTools package effectively. Here is the link to the tutorials: https://github.com/asreview/asreview-datatools/blob/master/Tutorials.md.

You could simple export the different sets from Zotero and use the `compose` option in Datatools to create a partly labeled dataset.

I hope these resources help you in your testing process. Keep up the good work and do let us know if it worked!

**Reply by Hanedinges** [reply]
I'm amazed by the speed of your reply, awesome!!!

**Reply by Juliately** [reply]
Good day, 

Is this link still available? 
https://asreview.readthedocs.io/en/latest/data_format.html

I am currently working on a SR in ASReview, but I'm having trouble with loading the partially labeled data that I want to use to train the model. 

thank you for your help!

**Reply by Rensvandeschoot** [reply]
This is the new link: https://asreview.readthedocs.io/en/latest/lab/data_format.html

**Comment by Juliately** [comment]
Thank you, I have managed 😊

Met vriendelijke groeten,

Julia Lyons

[55769_fullimage_handtekening 3]



Van: Rens van de schoot ***@***.***>
Verzonden: donderdag 12 juni 2025 08:55
Aan: asreview/asreview ***@***.***>
CC: Lyons, J.T. (Julia) ***@***.***>; Comment ***@***.***>
Onderwerp: Re: [asreview/asreview] Prelabeled data - 1. ris ->csv? 2. adding labels with columns? . (Discussion #1468)


This is the new link: https://asreview.readthedocs.io/en/latest/lab/data_format.html

—
Reply to this email directly, view it on GitHub<https://github.com/asreview/asreview/discussions/1468#discussioncomment-13442294>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BTO75F3Q5OP7NCULQV2HOXL3DEP47AVCNFSM6AAAAAB7C5RZWOVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTGNBUGIZDSNA>.
You are receiving this because you commented.Message ID: ***@***.***>

______________________________________________________
AmsterdamUMC disclaimer : www.amsterdamumc.org/nl/disclaimers.htm
